src/main/java/com/github/manolo8/darkbot/extensions/features/FeatureDefinition.java

```diff
@@ -162,4 +163,26 @@ public boolean canLoad() {
     public void addIssue(@NotNull String message, @NotNull String description, @NotNull Level level) {
         issues.add(message, description, PluginIssue.Level.values()[level.ordinal()]);
     }
+
+    /**
+     * @return true if exception cause was {@link FeatureException}
+     */
+    public boolean addIssue(Throwable e, String message, PluginIssue.Level level) {
```

Comment:
```suggestion
    public boolean addIssue(String message, Level level, Throwable e) {
```
It's convention for throwable to be the last param in logging libraries and such